templates/auction-listing-email.mjml

```diff
@@ -52,15 +52,19 @@
         <mj-divider border-color="#000000"></mj-divider>
 
         <mj-text font-size="16px" color="#000000" font-family="Red Hat Display">
-         Your seven day auction is set to begin at 3pm EST. We may email you for additional collateral if your item is especially high-value.
+          Your auction is pre-live. Click <a href="{{ encryptedDashboardUrl }}">here to view, edit, or delete</a> any of your drops. Do <strong>NOT</strong> share this link with anyone.
```

Comment:
Our internal (development) statuses are leaking into business context. Bidders and sellers have no real context of wtf "pre-live" is.

This is a tiny nit so I'm not gonna block it. Just watch out for future instances of leaking internal language to users that may have no idea wtf an internal term may mean.

Chris may pushback here when he sees this. At that point, we'll fix. If no pushback, it'll sneak on through.